test/factories/article.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :article do
+    title { Faker::Book.title }
+    body { Faker::Lorem.paragraph }
+    user_id { Faker::Number.between(from: 1, to: 10) }
```

Comment:
Could you try changing `user_id { Faker::Number.between(from: 1, to: 10) }` into `user { create :user }` ?

I think that should take care of the id here.

Reply:
I changed it and it works, and it is more readable that way.